### DISCUSSION TITLE
Project name PSA

### DISCUSSION THREAD

**Discussion by danielnieto** [eronka/culero-api#89] [Ideas]
"Culero" is a Spanish slang term that can be translated to "455h0l3" in English. It's a vulgar term often used to insult someone, implying they are mean, unpleasant, or contemptible. 

The exact meaning and level of offensiveness can vary by region and context.

It caught my attention when I saw the ad on reddit, and wanted to let you know that the project name is offensive in many Spanish speaking countries in case you're unaware.

**Comment by mariomac** [comment]
I was looking for LinkedIn alternatives and found this amazing project, and I was about to start the same discussion, just as an advice.

https://www.wordreference.com/es/en/translation.asp?spen=culero
